concepticondata/concepticon.tsv

```diff
@@ -3933,4 +3933,27 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3947	HONEYCOMB	Animals	A wax structure consisting of multiple small holes made by bees to store their honey.	Person/Thing	
 3948	SHOTGUN	Warfare and hunting	A long gun (used mainly for hunting) that shoots multiple small bullets at a time.	Person/Thing	
 3949	RED-FOOTED TORTOISE	Animals	A medium-sized species of tortoise (Chelonoidis Carbonarius), found in northern parts of South America.	Person/Thing	
-3950	LIBER (INNER BARK)	Agriculture and vegetation	The inner most layer of bark (particularly from a tree).	Person/Thing	
\ No newline at end of file
+3950	LIBER (INNER BARK)	Agriculture and vegetation	The inner most layer of bark (particularly from a tree).	Person/Thing	
+3951	PANDA	Animals	Also *giant panda*, a white-and-black mammal of the *Ailuropoda* family, native to eastern Asia, commonly referred to as a type of bear.	Person/Thing	
+3952	REFRIGERATOR	Modern world	A storage box, compartment or room to stock goods by means of mechanical refrigeration.	Person/Thing	
+3953	CHARITY	Social and political relations	A benevolent action in favor of the needy, or an activity or gift for the benefit of the public.	Property	
+3954	FRIENDSHIP (FEELING)	Emotions and values	A feeling of affection, good disposition, and sympathy experienced towards someone.	Property	
```

Comment:
Not sure if a friendship is a feeling. We don't have a similar concept yet, so FRIENDSHIP should be sufficient.

Reply:
Semantic Field: Social and political relations